source/slang/glsl.meta.slang

```diff
@@ -4802,36 +4804,34 @@ __generic<T : __BuiltinType>
 }
 
 ${{{{
-
-struct TypeClassSubclass {
+static const struct {
     const char* name;
     const char* classType;
     const char *subclassType;
     const char *suffix;
-    const int isFloat;
-}; 
-const TypeClassSubclass atomics[] =
+    const bool isFloat;
+} atomics[] =
```

Comment:
Please split the declaration of the struct from the declaration of the variable.
This type of expression is harder to read and easy to make a mistake.

Reply:
Please not use this complex syntax.
It is harder to read and easy to make a mistake.

Reply:
to copy how the code works in hlsl.meta I set the struct to the same `static const struct` format (for consistency purposes)